src/DwitTech.AccountService.Core/Services/UserService.cs

```diff
@@ -175,5 +179,20 @@ public async Task<bool> LogoutUser(string authHeader)
                 throw new Exception($"{ex}");
             }
         }
+
+        public async Task<bool> EditAccount(string authHeader, EditRequestDto editDto)
```

Comment:
I think this "authHeader" here can be named better. If it's a token, then it might as well be named "authToken" or something related.